installer/datafiles/linux.data

```diff
@@ -199,12 +199,15 @@ fi
 # Ensure omsagent is in the omiusers group, but leave omsagent as a group
 /usr/sbin/usermod -g omiusers omsagent 1> /dev/null 2> /dev/null
 
-%Postinstall_200
+%Postinstall_500
```

Comment:
Anurag asked for this change to allow us to be part of the SCOM Linux Agent. Currently the agent gets started after install even if it isn't onboarded.